colorbleed/maya/__init__.py

```diff
@@ -96,10 +97,16 @@ def _fn():
     safe_deferred(override_component_mask_commands)
 
 
+def before_save_check(return_code, _):
```

Comment:
Shouldn't this match naming like the other methods, the `_check` seems redundant looking at the other methods like `on_save`, `on_load`, etc. :)